src/cmd-push-container-manifest

```diff
@@ -95,6 +95,12 @@ def main():
         # Create/Upload the manifest list to the container registry
         manifest_info = create_and_push_container_manifest(
             args.repo, args.tags, images, args.v2s2)
+        if args.v2s2:
```

Comment:
I feel like some of the context from the commit message would be good as a comment here.

Reply:
Updated commit message and code with context!